### ISSUE TITLE
'ngInject' trigger an 'expected an assignment or function call' error

### ISSUE THREAD

**Issue by mfilotto** [palantir/tslint#423]
Hi,
I'm working on an angular app.
When I use ngInject for depency injection, I have the error 'expected an assignment or function call'
Is there any way to tell tslint to ignore 'ngInject'; instructions ?

**Comment by adidahiya** [comment]
It might help if you posted more code here.
Do you mean you have a statement that looks like this?

```
'ngInject';
```

You could disable tslint for that line using comments. What's the exact rule that's failing here? You can find out using the verbose output option. (I would check, but I'm not at a computer right now)

**Comment by mfilotto** [comment]
That's it, I have this statement in my code.
The error code I have on that line is 'expected an assignment or function call'

````` javascript
class NavbarController {
    currentDate: Date;
    constructor(translator) {
        'ngInject';
        this.currentDate = new Date();
    }
}
````

**Comment by leeavital** [comment]
It sounds like you're using `no-unused-expression`, which is meant to prevent no-ops like the one you have written here.

I think your best bet is to use comments to disable tslint around `ngInject`.
